packages/fxa-auth-server/package.json

```diff
@@ -69,6 +69,7 @@
     "hkdf": "0.0.2",
     "hot-shots": "^7.6.0",
     "i18n-abide": "0.0.26",
+    "intl": "1.2.5",
```

Comment:
Weird... this module was last released 4 years ago? Guess that just means it's stable?